lib/api/apiUtils/object/validRangeOnEmptyFile.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * validRangeForEmptyFile - Validate range request header on empty object
+ * @param {string} rangeHeader - range header from request
+ * which should be in form bytes=0-9
+ * @return {bool} object true if valid range, false if not
+ */
+export function validRangeOnEmptyFile(rangeHeader) {
+    if (rangeHeader.startsWith('bytes=') && rangeHeader.indexOf('-') > -1) {
+        const rangePortion = rangeHeader.replace('bytes=', '').split('-');
+        if (rangePortion.length === 2) {
+            // - if end value only, value has to be 0
+            // - else if start value only, value has to be a positive integer
+            // - else if start and end values, both values have to be positive
+            // integer and end value superior or equal to start value
+            if (rangePortion[0] === '' && parseInt(rangePortion[1], 10) === 0) {
```

Comment:
Here I m just checking that parseInt(rangePortion[1]) is equal to 0,  if parseInt(rangePortion[1]) is not a number it will return NaN.